docs/formats/coco_user_manual.md

```diff
@@ -0,0 +1,177 @@
+- [Panoptic Segmentation](https://cocodataset.org/#panoptic-2020) (`coco_panoptic`)
+- [Keypoint Detection](https://cocodataset.org/#keypoints-2020) (`coco_person_keypoints`)
+- [Stuff Segmentation](https://cocodataset.org/#stuff-2019) (`coco_stuff`)
+
+To make sure that the selected dataset has been added to the project, you can run
+`datum info`, which will display the project and dataset information.
+
+##  Export to other formats
+
+Datumaro can convert COCO dataset into any other format [Datumaro supports](../user_manual.md#supported-formats).
+To get the expected result, the dataset needs to be converted to formats
+that support the specified task (e.g. for panoptic segmentation - VOC, CamVID)
+There are few ways to convert COCO dataset to other dataset format:
+
+``` bash
+datum project import -f coco -i <path/to/coco>
+datum export -f voc -o <path/to/output/dir>
+# or
+datum convert -if coco -i <path/to/coco> -f voc -o <path/to/output/dir>
+```
+
+Some formats provide extra options for conversion.
+These options are passed after double dash (`--`) in the command line.
+To get information about them, run
+
+`datum export -f <FORMAT> -- -h`
+
+##  Export to COCO
+
+There are few ways to convert dataset to COCO format:
+
+``` bash
+# export dataset into COCO format from existing project
+datum export -p <path/to/project> -f coco -o <path/to/export/dir> \
+    -- --save-images
+# converting to COCO format from other format
+datum convert -if voc -i <path/to/voc/dataset> \
+    -f coco -o <path/to/export/dir> -- --save-images
+```
+
+Extra options for export to COCO format:
+- `--save-images` allow to export dataset with saving images
+(by default `False`);
+- `--image-ext IMAGE_EXT` allow to specify image extension
+for exporting dataset (by default `.png`);
+- `--segmentation-mode MODE` allow to specify save mode for instance segmentation:
+    - 'guess': guess the mode for each instance (using 'is_crowd' attribute as hint)
+    - 'polygons': save polygons( merge and convert masks, prefer polygons)
+    - 'mask': save masks (merge and convert polygons, prefer masks)
+(by default `guess`);
+- `--crop-covered` allow torop covered segments so that background objects
+segmentation was more accurate (by default `False`);
+- `--allow-attributes ALLOW_ATTRIBUTES` allow export of attributes
+(by default `True`);
+- `--reindex REINDEX` allow to assign new indices to images and annotations,
+useful to avoid merge conflicts (by default `False`);
+- `--merge-images` allow to save all images into a single directory
+(by default `False`);
+- `--tasks TASKS` allow to specify tasks for export dataset,
+by default Datumaro uses all tasks. Example:
+
+```bash
+datum import -o project -f coco -i <dataset>
+datum export -p project -f coco -- --tasks instances,stuff
+```
+
+## Particular use cases
+
+Datumaro supports filtering, transformation, merging etc. for all formats
```

Comment:
Add 2 examples: how to load the original format and how to create a custom dataset in this format. You can also add a reference to tests.

Reply:
Is it enough?